locale/moretrees.es.tr

```diff
@@ -211,10 +211,10 @@ Willow Tree Trunk Stair=Escalera de Tronco de Árbol de Sauce
 Yellow=Amarillo
 
 # init.lua
-Sapling=
-Tree=
-Wooden Planks=
-Leaves=
-Wooden Fence=
-Wooden Fence Rail=
-Wooden Fence Gate=
+Sapling=Retoño
+Tree=Puu
+Wooden Planks=Tablones
+Leaves=Hojas
+Wooden Fence=Riel de Valla
```

Comment:
Wooden Fence=Valla
or
Valla de Madera
if you want to specify that it is wooden.
An alternative word would be "cerca" but I think "Valla" is a better fit.

Reply:
Ok